### ISSUE TITLE
CGP: Additional location pages under "Locations" menu to subdomain pages 

### ISSUE THREAD

**Issue by alfredchu77** [mikedesigner/CGP#1]
For Cities include Shanghai, Beijing, Guangzhou, Suzhou, Hong Kong, Singapore in the pull down menu.

**Comment by mikedesigner** [comment]
Are you talking about the top main menu:
Change "contact" to "locations"
Locations

> Shanghai
> Beijing
> Guangzhou
> Suzhou
> Hong Kong
> Singapore

What will each of these pages look like ? Like the contact page ? I am closing creating new layouts so we dont run into development creep, which could be phase 2.

**Comment by alfredchu77** [comment]
This would be a jump to the section of our contact page with the address details.  But for HK and Singapore would be a separate about page.

**Comment by alfredchu77** [comment]
This would be a jump to the section of our contact page with the address details.  But for HK and Singapore would be a separate about page.

**Comment by alfredchu77** [comment]
This would be a jump to the section of our contact page with the address details.  But for HK and Singapore would be a separate about page.

**Comment by alfredchu77** [comment]
Yes, we have a menu that lists out our locations but Hong Kong and Singapore has their own links to a 1 pager.
